h/static/styles/site.scss

```diff
@@ -1,5 +1,4 @@
 // minimal bootstrap CSS
-@import './vendor/bootstrap/bootstrap';
 
```

Comment:
This bootstrap import will be removed by https://github.com/hypothesis/h/pull/3601 which should be merged first, so this pr won't have to remove this import